infrastructure/k8s/services/notify.yaml

```diff
@@ -0,0 +1,42 @@
+project_name: "taskcluster-notify"
+secrets:
+  aws_access_key_id: ".Values.awsAccessKeyId"
```

Comment:
```suggestion
  aws_access_key_id: ".Values.notifyAwsAccessKeyId"
```

Reply:
fixed in https://github.com/sciurus/taskcluster/pull/14/commits/09a577fa48a3b4b681be23532f4076d535326eef